Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -2778,3 +2778,37 @@ Function Set-AVSVSANClusterUNMAPTRIM {
         }
     }
 }
+
+<#
+    .DESCRIPTION
+     Set default Multipath Policy to Round Robin.
+
+    .PARAMETER DatastoreName
+     Datastore name
+
+    .EXAMPLE
+     Set-DatastoreMultipathingPolicy -DatastoreName "myDatastore"
+
+    .INPUTS
+     vCenter datastore name.
+
+    .OUTPUTS
+     None.
+#>
+function Set-DatastoreMultipathingPolicy {
+    [CmdletBinding()]
+    [AVSAttribute(10, UpdatesSDDC = $false, AutomationOnly = $true)]
```

Comment:
What automation is going to call this? 
Also, perhaps it should go into VMFS package.

Reply:
Hi Eugene,
1. It will be called while creating pure datastores.
2.  Yes, I have shifted it to VMFS.
 For further clarification regarding the script kindly contact - tyagi.prashant@microsoft.com, mchhibber@microsoft.com, we can get in call as well as this feature is required to be implemented on urgent basis.